src/bot/exts/dragonfly/dragonfly.py

```diff
@@ -582,6 +583,35 @@ async def before_scan_loop(self: Self) -> None:
         """Wait until the bot is ready."""
         await self.bot.wait_until_ready()
 
+    @commands.has_role(Roles.vipyr_security)
+    @commands.hybrid_command()
+    async def queue(self: Self, ctx: commands.Context, name: str, version: str) -> None:
+        """Add a package to the Dragonfly scan queue."""
+        try:
+            await self.bot.dragonfly_services.queue_package(name=name, version=version)
+            await ctx.send(f"Successfully queued package `{name} v{version}`")
+        except aiohttp.ClientResponseError as http_error:
+            status_code = http_error.status
+
+            if status_code == HTTPStatus.NOT_FOUND:
+                await ctx.send(f"Package `{name} v{version}` was not found on PyPI")
+
+            if status_code == HTTPStatus.CONFLICT:
+                scan_results: list[Package] = await self.bot.dragonfly_services.get_scanned_packages(
+                    name=name,
+                    version=version,
+                )
+                package = scan_results[0] if scan_results else None
+
+                if package:
+                    embed = _build_package_scan_result_embed(scan_results[0])
+                    await ctx.send(f"Package `{name} v{version}` has already been scanned.", embed=embed)
+                else:
+                    await ctx.send(f"Package `{name} v{version}` is already waiting to be scanned.")
```

Comment:
Just combine both these into one if statement. `package` was unused, also.
```suggestion
                if len(scan_results) != 0:
                    embed = _build_package_scan_result_embed(scan_results[0])
                    await ctx.send(f"Package `{name} v{version}` has already been scanned.", embed=embed)
                else:
                    await ctx.send(f"Package `{name} v{version}` is already waiting to be scanned.")
```